README.md

```diff
@@ -1,4 +1,16 @@
 wp-cli/ideas
 ============
 
-Have an idea for a WP-CLI command? [Create a new issue](https://github.com/wp-cli/ideas/issues/new) or [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
+### Have an idea for a WP-CLI command?**
+
+Feel free to [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) if you can find one that matches, or [create a new issue](https://github.com/wp-cli/ideas/issues/new) if no one has proposed the idea yet.
+
+### Curious as to how ideas become WP-CLI commands?
+
+The process generally works like this:
+
+0. **New issue is created.** Good issues include substantial detail into what the command might do, how it might work, and what value it would add to the ecosystem.
+1. **Some discussion takes place.** Issues are evaluated against [WP-CLI's philosophy](https://make.wordpress.org/cli/handbook/philosophy/), technical feasibility, and estimated impact. Everyone is free to weigh in; we value thoughtful consideration that introduces new viewpoints.
+2. **New WP-CLI commands are created.** In some cases they're written as standalone packages, and in others they're submitted as pull requests to existing repositories. In some cases they're produced by committers, and in others they're developed by members of the community. It's a pretty organic process.
```

Comment:
Maybe
2. Some development takes place. In some cases...
3. Iterate between 1. and 2. It's a pretty organic process.
4. A new WP-CLI command is born/created.